Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+standardPipeline ([timeout: 60], {
+  standardNode {
+    checkoutStage {}
+
+    stage('Update .version') {
+      sh "date +%Y%m%d-${env.BUILD_NUMBER} > .version"
+      echo "Set version to ${readFile '.version'}"
+    }
+
+    if (isMasterBranch()) {
+      def version = latestDockerTag repoName()
+      def tags = dockerBuildStage {}
+      autoPublishStage {
```

Comment:
Why not keep it simple like all the other images?

Reply:
I used this template before, just reuse it.
as I said this template is adding a date to docker image. So, if it's better to follow the simple one,I'll do that

Reply:
yes please. let's keep .version like all others. No reason to have a timestamp for this.